shell/components/ExplorerMembers.vue

```diff
@@ -324,7 +328,7 @@ export default {
         />
       </Tab>
       <Tab
-        v-if="canManageProjectMembers"
+        v-if="canManageProjectMembers && !isHarvester"
```

Comment:
Harvester want to display `Cluster Membership` only

Harvester:
![image](https://github.com/rancher/dashboard/assets/15041915/cc5b8b59-1f73-4792-b774-3db488437fef)

Rancher:
![image](https://github.com/rancher/dashboard/assets/15041915/b7f21c3f-be08-4c59-a8d2-1083092c0960)